frontend/src/pages/exploreApplication/EnableModal.tsx

```diff
@@ -115,19 +135,17 @@ const EnableModal: React.FC<EnableModalProps> = ({ selectedApp, onClose }) => {
               />
             </FormAlert>
           ) : null}
-          {selectedApp.spec.enable?.variables
-            ? Object.keys(selectedApp.spec.enable.variables).map((key, index) => (
-                <EnableVariable
-                  key={key}
-                  ref={index === 0 ? focusRef : undefined}
-                  label={selectedApp.spec.enable?.variableDisplayText?.[key] ?? ''}
-                  inputType={selectedApp.spec.enable?.variables?.[key] as TextInputTypes}
-                  helperText={selectedApp.spec.enable?.variableHelpText?.[key] ?? ''}
-                  validationInProgress={validationInProgress}
-                  updateValue={(value: string) => updateEnableValue(key, value)}
-                />
-              ))
-            : null}
+          {Object.keys(enable.variables).map((key, index) => (
```

Comment:
future

this looks like a simple update right now, just removed the condition, it works. didn't catch this before, but maybe a future update could look at `Object.entries` if we've got `enable.variables[key]` in there